enterprise/reporting/reporting.go

```diff
@@ -613,6 +613,12 @@ func (r *DefaultReporter) getTags(label string) stats.Tags {
 }
 
 func (r *DefaultReporter) Stop() {
+	r.syncersMu.RLock()
+	for _, syncer := range r.syncers {
+		_ = syncer.DbHandle.Close()
+	}
+	r.syncersMu.RUnlock()
+
```

Comment:
`DatabaseSyncer` method creates the handle, it should also be the one that needs to close it